db/sql/cassandra/plugin_impl_cassa.go

```diff
@@ -73,18 +82,28 @@ func (p *Plugin) AfterInit() error {
 		p.session = gockle.NewSession(session)
 	}
 
-	/* TODO Register for providing status reports (polling mode)
+	// Register for providing status reports (polling mode)
 	if p.StatusCheck != nil && p.session != nil {
 		p.StatusCheck.Register(core.PluginName(p.String()), func() (statuscheck.PluginState, error) {
-			_, _, err := p.Skeleton.NewBroker("/").GetValue(healthCheckProbeKey, nil)
+			broker := p.NewBroker()
```

Comment:
just do broker.Exec("readonly cassandra query that is runalbe in every schema without having extra objects")

Reply:
done

Reply:
If possbile try to use directly connection broker to not create new instance again

Reply:
done